my-house/src/app/TODO-list/page.tsx

```diff
@@ -1,9 +1,18 @@
-import styles from './page.module.scss'
+'use client'
```

Comment:
```suggestion
'use client'
```
dlaczego ta dyrektywa?

Reply:
a dobra czekaj, to next, okay ja sobie w kontekście CRA pomyślałem